hera_opm/mf_tools.py

```diff
@@ -133,31 +149,59 @@ def make_outfile_name(obsid, action, pol_list=[]):
         The obsid of the file.
     action : str
         The action corresponding to the output name.
-    pol_list : list of str
-        List denoting polarizations in files; if an empty list,
-        then all polarizations in a single file is assumed. (Default empty list)
 
     Returns
     -------
     outfiles : list of str
         A list of files that represent output produced for `action`
-        corresponding to `obsid`. For multiple polarizations, contains one
-        string per polarization. For one or no polarizations, just a list with
-        a single entry is returned.
+        corresponding to `obsid`.
 
     """
-    outfiles = []
-    if len(pol_list) > 1 and pol_list[0] is not None:
-        for pol in pol_list:
-            of = "{0}.{1}.{2}.out".format(obsid, action, pol)
-            outfiles.append(of)
-    else:
-        of = "{0}.{1}.out".format(obsid, action)
-        outfiles.append(of)
-    return outfiles
+    return [f"{obsid}.{action}.out"]
```

Comment:
Gotta love that this turned into a one-liner without having to worry about polarization 😄